RecoBTag/TensorFlow/python/pfDeepDoubleCvBJetTags_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+from pfDeepDoubleBvLJetTags_cfi import pfDeepDoubleBvLJetTags
+
+pfDeepDoubleCvBJetTags = pfDeepDoubleBvLJetTags.clone(
+    flavor = 'CvB',
+    graph_path = cms.FileInPath('RecoBTag/Combined/data/DeepDoubleX/94X/V01/DDCvB.pb'),
+    flav_table = cms.PSet(
+      probHbb = cms.vuint32(0),
+      probHcc = cms.vuint32(1)
```

Comment:
the issue(s) from the code review in this file are still not resolved.

please drop type specifications for all parameters which already exist.

IIUC, the optional parameters encoded in the fillDescriptions should be enough.

What about
```
pfDeepDoubleCvBJetTags = cms.EDProducer("DeepDoubleXTFJetTagsProducer",
flavor = 'CvB'
)
```

Reply:
modifying this file according to your suggestions results in 
`Exception Message:
python encountered the error: <type 'exceptions.TypeError'>
flavor does not already exist, so it can only be set to a CMS python configuration type`